### DISCUSSION TITLE
Can we build unikernel for Nodejs application using Unikraft?

### DISCUSSION THREAD

**Discussion by rahulc-globant** [unikraft/unikraft#557] [Q&A]
I need to create an application of NodeJS server using Unikraft.
But I am not able to get any leads on this through documentation. Is this supported?
If not, can we create a server using GO, if yes can we add some references here

**Comment by razvand** [comment]
@rahulc-globant, since we are already (viciously) using [the Unikraft Discord server](https://bit.ly/UnikraftDiscord) for discussions, meetings, brainstorming and jokes (mostly bad, we do what we can), we will close the Discussions tab. We will do this on Friday, March 31, 2023.

Please move any discussions or questions on [the Unikraft Discord server](https://bit.ly/UnikraftDiscord). See you all there! :handshake:
